src/components/Navigation.css

```diff
@@ -0,0 +1,85 @@
+.header-wrapper {
+  position: absolute;
```

Comment:
This seems strange, you have that empty `header` element on the page, which is residing perfectly nice and fine in the `grid`!